packages/foreman/nodejs-patternfly-react/nodejs-patternfly-react.spec

```diff
@@ -17,6 +18,8 @@ Requires: npm(classnames) >= 2.2.5
 Requires: npm(classnames) < 3.0.0
 Requires: npm(css-element-queries) >= 1.0.1
 Requires: npm(css-element-queries) < 2.0.0
+Requires: npm(lodash) >= 4.17.11
```

Comment:
Looks like we include 4.17.10 in our repo. Could you bump it as well? The same script can handle it. Can all be part of the same PR.